content/en/building/translations/overview.md

```diff
@@ -14,14 +14,14 @@ Given that CHT applications are used around the world, the Core Framework was de
 
 ## Key Concepts
```

Comment:
Similarly to the `TIP` you added in the `localizing` page, I suggest we also link somewhere on this page to the contributing translations page.  Repetitive, but better to ensure people find it if needed.

Reply:
👍  More linkage is better!